docs/src/resources/two-side-column.md

```diff
@@ -0,0 +1,519 @@
+<!Doctype html>
+<html>
+<title> Two side column </title>
+<body>
+    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+    <table border ="0">
+        <h1>Creating a Module</h1>
+        <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+            <td><b style="font-size:30px">Generated Verilog</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+    module foo (
+                    input  a,
+                    output b
+                )
+                assign b = a;
+            endmodule
+```
+
+</td>
+    <td>
+
+```scala mdoc
+    class Foo extends Module {
+    val a = Input(Bool())
+    val b = Output(Bool())
+    b := a
+    }
+```
+
+</td>
+    <td>Text comes here</td>
+         </tr>
+    <tr>
+<td>
+Text comes here
+</td>
+<td>
+
+```scala mdoc:invisible
+import Chisel.Queue
+import chisel3._
+import chisel3.util.DecoupledIO
+```
+
+```scala mdoc:silent
+class PassthroughGenerator(width: Int) extends Module {
+val io = IO(new Bundle {
+val in = Input(UInt(width.W))
+val out = Output(UInt(width.W))
+})
+io.out := io.in
+}
+println(getVerilogString(new PassthroughGenerator(10)))
+println(getVerilogString(new PassthroughGenerator(20)))
+```
+</td>
+<td>
+
+```
+module PassthroughGenerator(
+  input        clock,
+  input        reset,
+  input  [9:0] io_in,
+  output [9:0] io_out
+);
+  assign io_out = io_in; // @[main.scala 13:10]
+endmodule
+
+module PassthroughGenerator(
+input         clock,
+input         reset,
+input  [19:0] io_in,
+output [19:0] io_out
+);
+assign io_out = io_in; // @[main.scala 13:10]
+endmodule
+```
+</td>
+         </tr>
+    </table>
+</body>
+</html>
+
+#Parameterizing a Module
```

Comment:
```suggestion
# Parameterizing a Module
```

Make sure there are spaces after the `#` for all headings

Reply:
ok